uwblueprint/childrens-aid-society
# Case overview skeleton pr changes

## Notion ticket link

[Ticket Name](https://www.notion.so/uwblueprintexecs/Case-Overview-page-skeleton-a45ba23e6f15411f9c1012dc10f3588b?pvs=4)



## Implementation description
* Changes to PR review by Bowen and Jessica. Changes include fixing the design to meet the requirements of the figma document, improving code quality through the use of component based styling and removing unnecessary logic from the footer (caseOverviewFooter)



## Steps to test
1. Check figma documentation for design match up
2. Check code for mistakes in implementation 



## What should reviewers focus on?
* If the design is more suitable as per the figma document


## Checklist
- [ x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x ] I have run the appropriate linter(s)
- [ x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR
